cms/templates/component.html

```diff
@@ -8,22 +8,28 @@
     </div>
     <ul class="component-actions">
       <li class="action-item action-edit">
-        <a href="#" class="edit-button action-button">
+        <button class="btn-default edit-button action-button">
```

Comment:
We aren't using ``btn-default`` anywhere else in Studio, so I wouldn't use it in this PR either. Is this just to remove the button styling? I'd rather see us use the Pattern Library class ``btn`` if possible, but maybe the shim isn't available.